core/pom.xml

```diff
@@ -11,7 +11,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <artifactId>core</artifactId>
-
+  <properties>
```

Comment:
this was moved to the project file, please merge master

Reply:
done.